bundles/org.openhab.binding.linktap/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,223 @@
+		<state readOnly="false">
+			<options>
+				<option value="0">Off</option>
+				<option value="1">Instant</option>
+				<option value="2">Calendar</option>
+				<option value="3">Day</option>
+				<option value="4">Odd-even</option>
+				<option value="5">Interval</option>
+				<option value="6">Month</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="man-mode-type">
+		<item-type>Switch</item-type>
+		<label>Manual Watering</label>
+		<description>Manual watering mode status</description>
+		<category>Water</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="watering-type">
+		<item-type>Switch</item-type>
+		<label>Watering</label>
+		<description>Active watering status</description>
+		<category>Water</category>
+		<state readOnly="false"/>
+	</channel-type>
+
+	<channel-type id="rf-linked-type">
+		<item-type>Switch</item-type>
+		<label>RF Linked</label>
+		<description>Is the device RF linked</description>
+		<category>Switch</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="flm-linked-type">
+		<item-type>Switch</item-type>
+		<label>FLM Linked</label>
+		<description>The device has a included flow meter</description>
+		<category>Switch</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="is-leak-type">
+		<item-type>Switch</item-type>
+		<label>High Flow Detected</label>
+		<description>Unusually high flow rate detected alert</description>
+		<category>Alarm</category>
+		<state readOnly="false"/>
+	</channel-type>
+
+	<channel-type id="is-clog-type">
+		<item-type>Switch</item-type>
+		<label>Low Flow Detected</label>
+		<description>Unusually low flow rate detected alert</description>
+		<category>Alarm</category>
+		<state readOnly="false"/>
+	</channel-type>
+
+	<channel-type id="fall-status-type">
+		<item-type>Switch</item-type>
+		<label>Fallen Status</label>
+		<description>The device has fallen</description>
+		<category>Alarm</category>
+		<state readOnly="false"/>
+	</channel-type>
+
+	<channel-type id="fail-status-type">
+		<item-type>Switch</item-type>
+		<label>Shutdown Value Failed</label>
+		<description>The device has failed to close the valve</description>
+		<category>Alarm</category>
+		<state readOnly="false"/>
+	</channel-type>
+
+	<channel-type id="final-segment-type">
+		<item-type>Switch</item-type>
+		<label>Final ECO Segment</label>
+		<description>In ECO mode this is true when the final ON watering on segment is running</description>
+		<category>Switch</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="signal-level-type">
+		<item-type unitHint="%">Number:Dimensionless</item-type>
+		<label>Signal Level</label>
+		<description>Reception Signal Strength</description>
+		<category>QualityOfService</category>
+		<state readOnly="true" pattern="%.0f %%"/>
+	</channel-type>
+
+	<channel-type id="battery-level-type">
+		<item-type unitHint="%">Number:Dimensionless</item-type>
```

Comment:
I think before it didn't work somewhere in the UI without this - I will give it a few checks now in case I added the state pattern afterwards, and no longer required for a quirk somewhere.

Reply:
Yeah whatever it was it must have been before those addition's, I've removed it for the % as you say not needed. All the UI's I thought I saw the units missing in appear to be working fine with the state pattern.